events/SCA_swedish_events.txt

```diff
@@ -317,7 +317,13 @@ country_event = {
 				}
 			}
 		}
-		SWE = { country_event = SCA_sweden.9 }
+		SWE = {
```

Comment:
This should be in event sweden.9 instead as the effect of the option. This event has the effect tooltip.

Reply:
Ok, sure. I wasn't actually sure where the best place to put that code was. I will fix it when I can.